Translation-Chinese.md

```diff
@@ -5,7 +5,8 @@
 [Français](./Translation-French.md) ·
 [Русский](./Translation-Russian.md) ·
 [Deutsch](./Translation-German.md) ·
-[日本語](./Translation-Japanese.md)
+[日本語](./Translation-Japanese.md) ·
+[Italiano](./Translation-Italian.md)
```

Comment:
You seem to miss the dot here 😄 But it's ok, it's just nitpicks that will make the file look consistent.

Reply:
It's actually fine, dots are *between* words. So there should be no dot after the last one.

P.S. Probably a better formatting would be to move dots to the beginning of line (with no dot before the first word).